js/app.js

```diff
@@ -1 +1,189 @@
 'use strict';
+
+var store1 = {
+  name: 'First And Pike',
+  cookieAverage: 6.3,
+  hoursOpen: ['8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],
+  getCustomers: function(){
+    return randomCustomers(23,65);
+  },
+  cookiesSold: function(){
+    return Math.round(this.getCustomers() * this.cookieAverage);
+  },
+};
+
+var store2 = {
+  name: 'SeaTac Airport',
+  cookieAverage: 1.2,
+  hoursOpen: ['8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],
+  getCustomers: function(){
+    return randomCustomers(3,24);
+  },
+  cookiesSold: function(){
+    return Math.round(this.getCustomers() * this.cookieAverage);
+  },
+};
+
+var store3 = {
+  name: 'Seattle Center',
+  cookieAverage: 3.7,
+  hoursOpen: ['8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],
+  getCustomers: function(){
+    return randomCustomers(11,38);
+  },
+  cookiesSold: function(){
+    return Math.round(this.getCustomers() * this.cookieAverage);
+  },
+};
+
+var store4 = {
+  name: 'Capital Hill',
+  cookieAverage: 2.3,
+  hoursOpen: ['8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],
+  getCustomers: function(){
+    return randomCustomers(20,38);
+  },
+  cookiesSold: function(){
+    return Math.round(this.getCustomers() * this.cookieAverage);
+  },
+};
+
+var store5 = {
+  name: 'Alki',
+  cookieAverage: 4.6,
+  hoursOpen: ['8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],
+  getCustomers: function(){
+    return randomCustomers(2,16);
+  },
+  cookiesSold: function(){
+    return Math.round(this.getCustomers() * this.cookieAverage);
+  },
+};
+
+var stores = [store1,store2,store3,store4,store5];
+console.log(stores);
+
+function randomCustomers (min,max) {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+var parentElement = document.getElementById('sales');
+
+var article = document.createElement('article');
+parentElement.append(article);
+
+//Store 1
+var h2 = document.createElement('h2');
+h2.textContent = store1.name;
+article.appendChild(h2);
+
+var p = document.createElement('p');
+p.textContent = 'Sales by hour for the ' + store1.name + ' store.';
+article.appendChild(p);
+
+var ul = document.createElement('ul');
+article.appendChild(ul);
+
+var totalCookies = 0;
+
+for (var i = 0; i < store1.hoursOpen.length; i++) {
+  var li = document.createElement ('li');
+  li.textContent = store1.hoursOpen [i] + ': ' + store1.cookiesSold() + ' cookies';
```

Comment:
This is a really great solution to getting the data on the page. However, part of the requirements was to calculate the cookies sold per hour and store them in an array.